conf/messages.en

```diff
@@ -43,6 +45,10 @@ site.textarea.char_limit = (Limit is {0} characters)
 unauthorised.title = You can’t access this service with this account
 unauthorised.heading = You can’t access this service with this account
 
+technicalDifficulties.pageTitle = Sorry, we're experiencing technical difficulties
+technicalDifficulties.heading = Sorry, we're experiencing technical difficulties
+technicalDifficulties.message = Please try again in a few minutes.
+
```

Comment:
Content should be:
- Sorry, there is a problem with the service
- Try again in a few minutes. We did not save your answers.